groupsforsections.py

```diff
@@ -0,0 +1,85 @@
+import json
+import sys
+import logging
+import requests
+import urllib
+import utils
+import yaml
+
+
+def create_group_category():
+    logging.debug('create_group_category() Called')
+
+    url = canvas_url + '/api/v1/courses/' + course_id + '/group_categories?name=' + urllib.quote_plus(
+            group_category_name)
+    response = api_handler(url, 'Post')
+    if response is None:
+        logging.error('Api call for \'creating group categories\' is not successful')
+        sys.exit(1)
```

Comment:
I see you log the error here which is good, but does the program have to exit? Can something more be done to possibly recover? Is there any additional info that would be useful to log?